### ISSUE TITLE
Baubles lost on Critical Damage with Draconic Armor

### ISSUE THREAD

**Issue by OfficialStamper** [brandon3055/Draconic-Evolution#1124]
## Base information
* Minecraft version:  1.12.2
* Mod version:  Draconic-Evolution-1.12-2.3.11.290-universal.jar
* Minecraft Forge version:  forge-1.12.2-14.23.4.2707-universal.jar
* Mod Pack: (if applicable)  All the Mods 3 5.9.7b

When receiving critical damage whilst wearing full Draconic Armor, all baubles in the bauble slots are lost.  No other inventory items are lost.

No Errors or Warnings shown in logs, other than the generated CHAT message as follows:

`[18:12:40] [main/INFO] [net.minecraft.client.gui.GuiNewChat]: [CHAT] Warning Critical Damage!!!` 

Please let me know if you require addition information.

**Comment by HeroOfFire45** [comment]
Could be mod incompatibility or something.

**Comment by OfficialStamper** [comment]
Could be, but I do not know how to track the problem. Can I provide you with more information? What would help?

**Comment by brandon3055** [comment]
do you have any gravestone mods installed? If so try reproducing this issue with them disabled.

**Comment by OfficialStamper** [comment]
yes I do, ToManyGraves, I will restore from the backup when the problem happened and report back.  Just to let you know I am still with you on this.  Watch this space :)
